language/builtin.hlb

```diff
@@ -445,6 +445,12 @@ option::copy allowEmptyWildcard()
 # @return an option to change the owner of the copy path.
 option::copy chown(string owner)
 
+# Modifies the permissions of the copied files.
+#
+# @param filemode the new permissions of the file.
```

Comment:
`@param owner`

Reply:
Nice catch! Though it should be `@param filemode` and `int filemode`!